### DISCUSSION TITLE
Separate report, remove, and scan projects to circumvent API restrictions

### DISCUSSION THREAD

**Discussion by cjr60013255** [ThioJoe/YT-Spammer-Purge#954] [Ideas]
Would it be possible to separate keys/project in a different release branch so that there are three different keys used for the scan, report, and remove operations. So that each respective task has it's own 10,000 API limit, therefore as to not going against Google's policy (by having duplicate projects)?

**Comment by ThioJoe** [comment]
Unfortunately that would take a ton of work, so I doubt I'll be able to do that
